CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/SearchText.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package au.gov.asd.tac.constellation.views.conversationview;
+
+import au.gov.asd.tac.constellation.utilities.datastructure.Tuple;
+import au.gov.asd.tac.constellation.utilities.text.StringUtilities;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javafx.scene.control.TextField;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+/**
+ * The SearchText class provides static helper methods to search a body of text for some given text, and related information.
+ * 
+ * @author sol695510
+ */
+public class SearchText {
+    
+    private static final Logger LOGGER = Logger.getLogger(SearchText.class.getName());
+    private static int logCounter = 1;
+    
+    /**
+     * Returns the number of times the text typed in the search bar is found in the messages currently being displayed in the view.
+     * 
+     * @param messages A list of ConversationMessage messages.
+     * @return The number of matches of the searched text found in the messages.
+     */
+    protected static int getSearchHits (final List<ConversationMessage> messages) {
+        final TextField searchText = ConversationBox.searchBubbleTextField;
+        int hits = 0;
+        
+        if (!searchText.getText().isEmpty()) {
+            for (final ConversationMessage message : messages) {
+                for (final ConversationContribution contribution : message.getVisibleContributions()) {
+                    hits += (StringUtilities.searchRange(contribution.getText(), searchText.getText())).size();
+                    
+                    LOGGER.log(Level.INFO, searchText.getText()
+                            + "|" + String.valueOf(logCounter)
+                            + "|" + contribution.getText()
+                            + "|" + String.valueOf(hits));
```

Comment:
Do we really need this logged everytime? Can this be changed from `Level.INFO` to `Level.FINE`?